src/docfx/config/Config.cs

```diff
@@ -62,6 +64,7 @@ internal sealed class Config
         /// The extend file addresses
         /// The addresses can be absolute url or relative path
         /// </summary>
+        [JsonConverter(typeof(OneOrManyConverter))]
```

Comment:
Is this OneOrManyConverter only working for string type?

Reply:
works for arrays of any type